addons/vagrant/extra_vars_branches.yml

```
@@ -0,0 +1,24 @@
+---
+# Add branches repository
+packetfence_install__centos_release_rpm: 'http://packetfence.org/downloads/PacketFence/RHEL7/packetfence-release-7.branches.noarch.rpm'
+packetfence_install__centos:
+  repos:
+    - packetfence-devel
+    - packetfence-branches
+
+packetfence_install__deb:
+  repos:
+    - debian
+    - debian-devel
+    - debian-branches
+
+# Replace by *exact* name of your package
+packetfence_install__centos_packages:
+   - packetfence-9.3.9-20200124160156.113848061.0007.fix~5075.el7
```

Should this be hardcoded in the config ?

---

`extra_vars_branches.yml` is like an example file that allow you to override, on CLI, some inventory variables.

I created this file in order to edit it by hand when I want to install branch specific package.

Let's me know if that makes sense for you

---

In fact, it makes more sense to replace these values by a lookup on environment variables.
```suggestion
packetfence_install__centos_packages: {{ lookup("env", "CENTOS_PKG_NAME") }}
```
I just need to convert this to a list

---

Done